test/unit/player.test.js

```diff
@@ -1107,13 +1107,22 @@ QUnit.test('play promise should resolve to native value if returned', function(a
 
 QUnit.test('should throw on startup no techs are specified', function(assert) {
   const techOrder = videojs.options.techOrder;
+  const fixture = document.getElementById('qunit-fixture');
 
   videojs.options.techOrder = null;
   assert.throws(function() {
-    videojs(TestHelpers.makeTag());
+    const tag = TestHelpers.makeTag();
+
+    fixture.appendChild(tag);
+
+    videojs(tag);
   }, 'a falsey techOrder should throw');
 
   videojs.options.techOrder = techOrder;
+
+  while (fixture.firstChild) {
```

Comment:
I don't think we need to do this. qunit should be taking care to empty out qunit-fixture element after a test is run.

Reply:
good point